src/App.js

```diff
@@ -1,27 +1,76 @@
-import React from 'react'
+import React, { useState } from 'react'
 import Nav from './Nav'
 import Homepage from './Homepage'
 import Login from './Login'
 import About from './About'
 
 import { Route, Switch } from 'react-router-dom';
+import { gql, useQuery, useMutation } from '@apollo/client';
+
 const App = () => {
 
+const [user, setUser] = useState({})
+
```

Comment:
Try to keep spacing the same throughout the files. This can be refactored towards the end but something we should be aware of

Reply:
Thank you for the reminder to continue using clean code.